### PR TITLE
New license and copyright rule

### LINKED ISSUE #2798: two files with same license and copyright are classified differently (one erroneously)
### Description

two files with the exactly the same copyright and license header are classified different (and one erroneously).

```
==> toolchain.txt <==
# Copyright lowRISC contributors.
# Licensed under the Apache License, Version 2.0, see LICENSE for details.
# SPDX-License-Identifier: Apache-2.0

==> yum-requirements.txt <==
# Copyright lowRISC contributors.
# Licensed under the Apache License, Version 2.0, see LICENSE for details.
# SPDX-License-Identifier: Apache-2.0
```

csv results:

```
$ cat scancode-report.csv 
Resource,type,percentage_of_license_text,scan_errors,license_expression,license__key,license__score,license__name,license__short_name,license__category,license__is_exception,license__is_unknown,license__owner,license__homepage_url,license__text_url,license__reference_url,license__scancode_text_url,license__scancode_data_url,license__spdx_license_key,license__spdx_url,start_line,end_line,matched_rule__identifier,matched_rule__license_expression,matched_rule__licenses,matched_rule__referenced_filenames,matched_rule__is_license_text,matched_rule__is_license_notice,matched_rule__is_license_reference,matched_rule__is_license_tag,matched_rule__is_license_intro,matched_rule__has_unknown,matched_rule__matcher,matched_rule__rule_length,matched_rule__matched_length,matched_rule__match_coverage,matched_rule__rule_relevance,package__type,package__namespace,package__name,package__version,package__qualifiers,package__subpath,package__primary_language,package__description,package__release_date,package__homepage_url,package__download_url,package__size,package__sha1,package__md5,package__sha256,package__sha512,package__bug_tracking_url,package__code_view_url,package__vcs_url,package__copyright,package__license_expression,package__declared_license,package__notice_text,package__root_path,package__contains_source_code,package__extra_data,package__purl
toolchain.txt,file,10.37,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
toolchain.txt,,,,apache-2.0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
toolchain.txt,,,,apache-2.0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
toolchain.txt,,,,,apache-2.0,73.33,Apache License 2.0,Apache 2.0,Permissive,False,False,Apache Software Foundation,http://www.apache.org/licenses/,http://www.apache.org/licenses/LICENSE-2.0,https://scancode-licensedb.aboutcode.org/apache-2.0,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.LICENSE,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.yml,Apache-2.0,https://spdx.org/licenses/Apache-2.0,2,2,apache-2.0_464.RULE,apache-2.0,apache-2.0,LICENSE,,True,,,,,3-seq,15,11,73.33,100.00,,,,,,,,,,,,,,,,,,,,,,,,,,,
toolchain.txt,,,,,apache-2.0,100.00,Apache License 2.0,Apache 2.0,Permissive,False,False,Apache Software Foundation,http://www.apache.org/licenses/,http://www.apache.org/licenses/LICENSE-2.0,https://scancode-licensedb.aboutcode.org/apache-2.0,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.LICENSE,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.yml,Apache-2.0,https://spdx.org/licenses/Apache-2.0,3,3,spdx-license-identifier: apache-2.0,apache-2.0,apache-2.0,,,,,True,,,1-spdx-id,6,6,100.00,100.00,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,file,13.95,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,apache-2.0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,unknown-license-reference,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,apache-2.0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,,apache-2.0,100.00,Apache License 2.0,Apache 2.0,Permissive,False,False,Apache Software Foundation,http://www.apache.org/licenses/,http://www.apache.org/licenses/LICENSE-2.0,https://scancode-licensedb.aboutcode.org/apache-2.0,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.LICENSE,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.yml,Apache-2.0,https://spdx.org/licenses/Apache-2.0,2,2,apache-2.0_463.RULE,apache-2.0,apache-2.0,LICENSE,,True,,,,,2-aho,10,10,100.00,100.00,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,,unknown-license-reference,100.00,Unknown License file reference,Unknown License reference,Unstated License,False,True,Unspecified,,,https://scancode-licensedb.aboutcode.org/unknown-license-reference,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/unknown-license-reference.LICENSE,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/unknown-license-reference.yml,LicenseRef-scancode-unknown-license-reference,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/unknown-license-reference.LICENSE,2,2,unknown-license-reference_283.RULE,unknown-license-reference,unknown-license-reference,LICENSE,,,True,,,True,2-aho,4,4,100.00,100.00,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,,apache-2.0,100.00,Apache License 2.0,Apache 2.0,Permissive,False,False,Apache Software Foundation,http://www.apache.org/licenses/,http://www.apache.org/licenses/LICENSE-2.0,https://scancode-licensedb.aboutcode.org/apache-2.0,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.LICENSE,https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/apache-2.0.yml,Apache-2.0,https://spdx.org/licenses/Apache-2.0,3,3,spdx-license-identifier: apache-2.0,apache-2.0,apache-2.0,,,,,True,,,1-spdx-id,6,6,100.00,100.00,,,,,,,,,,,,,,,,,,,,,,,,,,,
yum-requirements.txt,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,pypi,,,,,,Python,,,,,,,,,,,,,,,,,yum-requirements.txt,,,
```

### How To Reproduce

On Ubuntu 20.04.2 LTS, running scancode 30.1.0 installed with pip3

bash script below:

```bash
$ cat ./scancode-bug1 
#!/usr/bin/env bash
# INPUT(S):
# $1: URL of git repo

rundatetime=$(date -u +%Y%m%dT%H%M%SZ)
tmpdir=$(mktemp -d -t oss-scancode-XXXXXXXX)
ossscanwebdir=/proj/web/infra/oss-check/$0
outdir="$PWD"
mkdir -p "$outdir"
pushd "$tmpdir" || exit
curl -O https://raw.githubusercontent.com/lowRISC/opentitan/master/yum-requirements.txt
curl -O https://raw.githubusercontent.com/lowRISC/opentitan/master/toolchain.txt
head -n 3 *
scancode -l -c -p --strip-root --html "$outdir"/scancode-report.html --json "$outdir"/scancode-report.json --csv "$outdir"/scancode-report.csv .
```

### PR DESCRIPTION
**Pull request by adityasangave**
Fixes #2798 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

**Comment by adityasangave** [comment]
@pombredanne review, any changes ?

File: src/cluecode/copyrights.py [hunk 1]
```diff
@@ -2562,6 +2562,9 @@ def build_detection_from_node(
     #holder is Tim Hudson (tjh@mincom.oz.au).
     COPYRIGHT: {<HOLDER> <JUNK> <NAME-EMAIL>}       #83001
 
+    # Copyright lowRISC contributors.
+    COPYRIGHT: {<COPY> <NN> <CONTRIBUTORS>}
+
```
Comment (pombredanne) [inline src/cluecode/copyrights.py:2567]: That's great! do you mind to add a test for this too?

**Review by pombredanne — CHANGES_REQUESTED** [review]
See https://github.com/nexB/scancode-toolkit/blob/develop/tests/cluecode/cluecode_test_utils.py and https://github.com/nexB/scancode-toolkit/tree/develop/tests/cluecode/data/copyrights for pairs of test file/expected results in YAML.

File: src/cluecode/copyrights.py (hunk 1 above)
Comment (adityasangave) [reply src/cluecode/copyrights.py:2567]: Sure thing :)

**Review by pombredanne — APPROVED** [review]
Looking great!
Thanks. Merging

**Pull request merged**